config.js

```diff
@@ -14,8 +14,10 @@
 // user is a global object provided by the Refocus server
 /* eslint no-process-env: 0 */
 const DEFAULT_PORT = 5000;
+const botName = require('./package.json').name;
 
 module.exports = {
+  botName,
```

Comment:
what's the reason for that? is the idea to use it somewhere?

Also (which is not a problem) but I saw in the other PR you renaming botName to name... why here is botName? is there a convention for that?

Reply:
plan to be used for events that are posted by bots that dont use a bot action as currently there is no way for them to add their botname to the event with our function. Renamed in other prs mainly for backwards compatability as now once all bots have this we should be able to source a botsname from the config not rely on someone to pass it into a function